controllers/grid/SubmissionListGridHandler.inc.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/**
+ * @file controllers/grid/SubmissionListGridHandler.inc.php
+ *
+ * Copyright (c) 2013-2018 Simon Fraser University
+ * Copyright (c) 2003-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class SubmissionListGridHandler
+ * @ingroup controllers_grid_sword
+ *
+ * @brief Handle submission list grid requests.
+ */
+
+import('lib.pkp.classes.controllers.grid.GridHandler');
+//import('plugins.generic.sword.controllers.grid.SubmissionListGridRow');
+import('lib.pkp.classes.controllers.grid.GridRow');
+import('plugins.generic.sword.controllers.grid.SubmissionListGridCellProvider');
+
+class SubmissionListGridHandler extends GridHandler {
```

Comment:
Likewise, is it possible to reuse an existing grid?

Reply:
please point me to a suitable one and I will make the changes.